pkg/service/sip.go

```diff
@@ -390,3 +391,64 @@ func (s *SIPService) CreateSIPParticipantRequest(ctx context.Context, req *livek
 	}
 	return rpc.NewCreateSIPParticipantRequest(projectID, callID, host, wsUrl, token, req, trunk)
 }
+
+func (s *SIPService) TransferSIPParticipant(ctx context.Context, req *livekit.TransferSIPParticipantRequest) (*emptypb.Empty, error) {
+	log := logger.GetLogger().WithValues("roomName", req.RoomName, "participantIdentity", req.ParticipantIdentity)
```

Comment:
we use `room` and `participant` as keys for these everywhere. Can we change it here?